base/abstractarraymath.jl

```diff
@@ -200,31 +200,61 @@ function repmat(a::AbstractVector, m::Int)
     return b
 end
 
-# Generalized repmat
-function repeat{T}(A::AbstractArray{T};
-                   inner::Array{Int} = ones(Int, ndims(A)),
-                   outer::Array{Int} = ones(Int, ndims(A)))
+"""
+    repeat(A::AbstractArray; inner=(), outer=())
```

Comment:
OK, I've put the actual signature in the docs, that's the most obvious solution. Should be good to go now.